### PR TITLE
Improved readme and added debug statements

### PR DESCRIPTION
**Pull request by AgarFu**
### Description

Improved the example in the README.md file and added some debug logs to help with troubleshooting

Thanks for contributing this Pull Request. Make sure that you send in this Pull Request to the `master` branch of this repository, add a brief description, and tag the relevant issue(s) and PR(s) below.

- Relevant Issues :

- What Changed and Why? :

- Type of change :
  - [ ] New feature
  - [ ] Bug fix for existing feature
  - [ ] Code quality improvement
  - [x] Addition or Improvement of tests
  - [x] Addition or Improvement of documentation

- Other:
  - [ ] Add unit tests
  - [ ] Add documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,16 +66,19 @@ class PreScale(Transformation):
...
 
-            for obj in file:
+            min_replicas = None
+            for obj in objects:
                 if obj['kind'] != 'HorizontalPodAutoscaler':
                     continue
 
                 if obj['spec']['minReplicas'] != obj['spec']['maxReplicas']:
+                    min_replicas = obj['spec']['minReplicas']
                     obj['spec']['maxReplicas'] = obj['spec']['minReplicas']
```
Comment (jnevelson) [inline README.md:78]: Should this be set to `min_replicas` then?

File: gordian/repo.py [hunk 2]
```diff
@@ -57,7 +62,8 @@ def get_objects(self, filename, klass=None):
 
     def get_files(self):
         if not self.files:
-            contents = self._repo.get_contents("")
+            logger.debug(f'Getting repo content')
+            contents = self._repo.get_contents('/')
```
Comment (jnevelson) [inline gordian/repo.py:66]: Is `get_contents('/')` the same as `get_contents('')`?

File: README.md (hunk 1 above)
Comment (AgarFu) [reply README.md:78]: 👍

File: gordian/repo.py (hunk 2 above)
Comment (AgarFu) [reply gordian/repo.py:66]: oops I didn't want to change that

Comment (AgarFu) [reply gordian/repo.py]: It is, I checked the library implementation, when the argument is an empty string it internally uses '/', but I just reverted it anyways

**Comment by AgarFu** [comment]
Merging and cutting a new release

**Pull request merged**